### PR TITLE
Stopwatch with HTML and JS (Workshop Bounty)

### PR DESCRIPTION
**Pull request by faisalsayed10**
Criteria:

- [x] Help club members produce a compelling and real project in under 20 minutes
Based on the test meeting, people were able to complete this workshop under 20 minutes.

- [x] Are very easy to hack on for long periods of time after completion
Many examples are listed on the end of the workshop as to how people can extend their project and learn more things.

- [x] Have instructive and compelling images, GIFs, and visuals
This workshop do contains instructive and compelling images, GIFs, and visuals.

- [x] Use repl.it or a similar service that allows members to begin coding with one click.
Repl.it is used for this workshop.

- [x] Link unknown concepts to external sources.
Linked many complex concepts such as var-let-const, querySelector, setInterval, clearInterval etc to useful sources from where people can learn more on it.

- [x] End with at least 3 demos of how the project can be expanded, with live links and code links for each demo. 
Have provided 5 demo examples to which people can refer and get an idea on how it can be expanded.

Good Workshop Guidelines:

- [x] Workshop that meets all of the "Good workshops" guidelines outlined above.

- [x] 1-2 minute video clip from your test club meeting showing what members built.
Here's the video clip from the demo meeting!
Link: https://hackclub.slack.com/archives/C01A9V5LPD0/p1602512151010300

- [x] All assets (images, PDFs, and any other binaries) must be hosted using the #cdn channel on the Hack Club Slack for posterity and to cut down on repository size.
All assets are hosted using the #cdn channel.

- [x] All workshops must be in English at native-level proficiency. Phrasing and grammar must be perfect.

- [x] You must be either a registered Hack Club leader or an active member of the Slack.
Club leader @ St. Mary's High School and also an active member @ slack.

### CODE REVIEW

**Comment by MatthewStanciu** [comment]
Fayd, omg, this is really good

**Comment by faisalsayed10** [comment]
Thank you!

File: workshops/stopwatch/README.md [hunk 1]
```diff
@@ -0,0 +1,480 @@
...
+### 2) HTML
+
+We need to add some more HTML lines to build our stopwatch. We will write all our HTML code inside the `container` div. Define a `<div>` tag with a class of `main` below the `<h1>` tag.
+
+```html
+<h1>Stopwatch</h1>
+<div class="main"></div>
+```
+
+Inside the `main` div, we will add a `<p>` tag with a class of `time` and seperate `<span>` tags for minutes, seconds and miliseconds respectively. Confusing? You'll get it in a minute.
```
Comment (bezlin6mechminerz) [inline workshops/stopwatch/README.md:54]:
```suggestion
Inside the `main` div, we will add a `<p>` tag with a class of `time` and separate `<span>` tags for minutes, seconds, and milliseconds respectively. Confusing? You'll get it in a minute.
```

Comment (bezlin6mechminerz) [reply workshops/stopwatch/README.md:54]: Just went through find this typo :)

File: workshops/stopwatch/README.md [hunk 2]
```diff
@@ -0,0 +1,480 @@
...
+
+Now, if you are wondering what is a `const` and what is a `querySelector()`, let me clear all your doubts!
+
+Mainly, there are 3 ways of defining a variable in JavaScript.
+
+1. `var`
+2. `let`
+3. `const`
+
+We will ignore `var` for now as we won't use it. If we define a variable using `let` then we can change its value anytime in the code afterwards. But, a `const` variable's value always remains constant and it will never change. Even if you try to change it, the console will throw you errors.
```
Comment (lachlanjc) [inline workshops/stopwatch/README.md:130]:
```suggestion
We will ignore `var` for now as we won't use it. If we define a variable using `let`, then we can change its value anytime in the code afterwards. Meanwhile, a `const` (“constant”)’s value always remains constant and cannot be changed—if you try to change it, JavaScript will show you an error in the console.
```

File: workshops/stopwatch/README.md [hunk 3]
```diff
@@ -0,0 +1,480 @@
+---
+name: 'Stopwatch'
+description: 'Build a simple stopwatch with HTML, CSS and JavaScript.'
+author: '@faisalsayed10'
```
Comment (lachlanjc) [inline workshops/stopwatch/README.md:4]:
```suggestion
author: '@faisalsayed10'
img: 'https://cloud-ijsj5lfdg.vercel.app/0image.png'
```

**Review by lachlanjc — APPROVED** [review]
Honestly, this is absolutely fantastic & I’m good to merge after that first suggestion. Great work here.

**Comment by faisalsayed10** [comment]
Thanks! Did that change : )

**Pull request merged**